README.md

```diff
@@ -1,5 +1,55 @@
 # Actions workflow run
 
+This GitHub Action will allow you to easily execute Chassy workflows within
+your automation pipelines.
+
+# Environment
+
+In addition to any configuration options, you also must have `CHASSY_TOKEN` defined within the environment.
+This is a secret value and as such should be stored within your repository's or organization's GitHub secrets.
+This value is what allows Chassy to authorize your workflow execution and prevents strangers from executing
+workflows that aren't theirs. It is quite a long string encoded in base64.
+
+| Variable       | Description                                           |
+| -------------- | ----------------------------------------------------- |
+| `CHASSY_TOKEN` | Authentication token for automated workflow execution |
+
+If `CHASSY_TOKEN` isn't defined, the action will fail to execute the workflow.
+
+# Configuration
+
+Each of these options can be used in the `with` section when you call this action.
+
+| Configuration        | Description                                                      | Type           | Default            |
+| -------------------- | ---------------------------------------------------------------- | -------------- | ------------------ |
+| `workflowId`         | ID of workflow you wish to execute                               | `string`       | **NONE. Required** |
+| `sync`               | Await completion of workflow execution                           | `boolean`      | `true`             |
+| `parameters`         | User-defined parameters for workflow (JSON format)               | `string`       | `'{}'`             |
+| `backendEnvironment` | Selects which chassy backend to use (`'PROD' | 'STAGE' | 'DEV'`) | `string union` | `'PROD'`           |
```

Comment:
this should not be exposed in the documentation. It is an internal detail.

Reply:
Agreed. Removed